libs/ngworker/lumberjack/console-driver/src/lib/log-drivers/lumberjack-console.driver.ts

```diff
@@ -1,40 +1,46 @@
 import { Inject, Injectable } from '@angular/core';
 
-import { LumberjackLogDriver, LumberjackLogDriverConfig } from '@ngworker/lumberjack';
+import {
+  LumberjackLogDriver,
+  LumberjackLogDriverConfig,
+  LumberjackLogDriverLog,
+  LumberjackLogPayload,
+} from '@ngworker/lumberjack';
 
 import { lumberjackConsoleDriverConfigToken } from '../configuration/lumberjack-console-driver-config.token';
 import { LumberjackConsole } from '../console/lumberjack-console';
 import { lumberjackConsoleToken } from '../console/lumberjack-console.token';
 
 @Injectable()
-export class LumberjackConsoleDriver implements LumberjackLogDriver {
+export class LumberjackConsoleDriver<TPayload extends LumberjackLogPayload | void = void>
+  implements LumberjackLogDriver<TPayload> {
   constructor(
     @Inject(lumberjackConsoleDriverConfigToken) public config: LumberjackLogDriverConfig,
     @Inject(lumberjackConsoleToken) private console: LumberjackConsole
   ) {}
 
-  logCritical(formattedLog: string): void {
-    this.console.error(formattedLog);
+  logCritical({ formattedLog, log }: LumberjackLogDriverLog<TPayload>): void {
+    this.console.error(formattedLog, log.payload || '');
   }
 
-  logDebug(formattedLog: string): void {
-    this.console.debug(formattedLog);
+  logDebug({ formattedLog, log }: LumberjackLogDriverLog<TPayload>): void {
+    this.console.debug(formattedLog, log.payload || '');
   }
 
-  logError(formattedLog: string): void {
-    this.console.error(formattedLog);
+  logError({ formattedLog, log }: LumberjackLogDriverLog<TPayload>): void {
+    this.console.error(formattedLog, log.payload || '');
```

Comment:
We should show that the payload is `undefined`, not empty string.

```suggestion
    this.console.error(formattedLog, log.payload);
```